src/pages/FormTransition.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect } from "react";
+import { useNavigate, useOutletContext, useParams } from "react-router-dom";
+import SplashScreen from "../components/SplashScreen";
+import { LayoutContext } from "./Layout";
+import PaniniCreator from "./PaniniCreator";
+
+export default function FormTransition() {
+  const { action } = useParams();
+  const { userStep, updateUserStep, setOrderData } = useOutletContext() as LayoutContext;
+  // Because this page is used only for animation purposes, on each visit after transition is done, navigate user to proper page.
+  const navigate = useNavigate();
```

Comment:
**issue:** You don't need to wrap it into other page - you nested 2 pages here in component which is not exactly a page. You don't need steps - just operate on routes.